Friendly-plans/app/src/androidTest/java/pg/autyzm/friendly_plans/manager_app/StepListFragmentTest.java

```diff
@@ -80,13 +80,13 @@ public void whenTaskHasNoStepsExpectNoStepsToBeDisplayed() {
     }
 
     @Test
-    public void whenSaveAndFinishClickedMainMenuShouldBeDisplayed() {
+    public void whenSaveClickedTaskCreateActivityShouldBeDisplayed() {
         long taskId = taskTemplateRule.createTaskWithSteps(TASK_NAME_1, STEP_NAME_1, STEP_NAME_2);
         openStepsListFragment(taskId);
```

Comment:
I just realized that we have copied setUp logic here - and we have it in all test methods :D could you please remove those two lines from all test methods? :)